apps/web/src/components/transactions/TxDetails/Summary/index.tsx

```diff
@@ -75,98 +73,86 @@ const Summary = ({ txDetails, defaultExpanded = false, hideDecodedData = false }
-                {gasPrice}
-              </TxDataRow>
-              <TxDataRow datatestid="tx-gas-token" title="gasToken:">
-                {generateDataRowValue(gasToken, 'hash', true)}
-              </TxDataRow>
-              <TxDataRow datatestid="tx-refund-receiver" title="refundReceiver:">
-                {generateDataRowValue(refundReceiver, 'hash', true)}
-              </TxDataRow>
-              {confirmations?.map(({ signature }, index) => (
-                <TxDataRow datatestid="tx-signature" title={`Signature ${index + 1}:`} key={`signature-${index}:`}>
-                  {generateDataRowValue(signature, 'rawData')}
-                </TxDataRow>
-              ))}
-
-              <Box mt={1}>
-                <TxDataRow datatestid="tx-raw-data" title="Raw data:">
-                  {generateDataRowValue(txData.hexData, 'rawData')}
-                </TxDataRow>
-              </Box>
-            </Box>
-          )}
-        </>
+      {!defaultExpanded && (
+        <Link
+          data-testid="tx-advanced-details"
+          className={css.buttonExpand}
+          onClick={toggleExpanded}
+          component="button"
+          variant="body1"
+        >
+          Advanced details
+        </Link>
       )}
-    </>
-  )
-}
 
-export default Summary
+      {expanded && (
+        <Box mt={2}>
+          {!isCustom && !hideDecodedData && <DecodedData txData={txData} toInfo={txData?.to} />}
 
-export const PartialSummary = ({ safeTx }: { safeTx: SafeTransaction }) => {
-  const txData = safeTx.data
-  const { safeAddress, safe } = useSafeInfo()
-  const safeTxHash = useMemo(() => {
-    if (!safe.version) return
-    try {
-      return calculateSafeTransactionHash(safeAddress, safeTx.data, safe.version, BigInt(safe.chainId))
-    } catch (e) {
-      logError(Errors._809, e)
-    }
-  }, [safe.chainId, safe.version, safeAddress, safeTx.data])
-  return (
-    <>
-      {safeTxHash && (
-        <SafeTxHashDataRow safeTxHash={safeTxHash} safeTxData={safeTx.data} safeVersion={safe.version as SafeVersion} />
-      )}
-      <TxDataRow datatestid="tx-safe-gas" title="safeTxGas:">
-        <SafeTxGasForm />
-      </TxDataRow>
-      <TxDataRow datatestid="tx-base-gas" title="baseGas:">
-        {txData.baseGas}
-      </TxDataRow>
-      <TxDataRow datatestid="tx-refund-receiver" title="refundReceiver:">
-        {generateDataRowValue(txData.refundReceiver, 'hash', true)}
-      </TxDataRow>
+          <Typography fontWeight="bold" pb={1}>
+            Transaction data
+          </Typography>
 
-      <Box mt={1}>
-        <TxDataRow datatestid="tx-raw-data" title="Raw data:">
-          {generateDataRowValue(txData.data, 'rawData')}
-        </TxDataRow>
-      </Box>
+          <TxDataRow datatestid="tx-to" title="to:">
+            {generateDataRowValue(safeTxData.to, 'address', true)}
+          </TxDataRow>
+
+          <TxDataRow datatestid="tx-to" title="value:">
```

Comment:
This should probably be:
```suggestion
          <TxDataRow datatestid="tx-value" title="value:">
```